tf2_gnn/layers/nodes_to_graph_representation.py

```diff
@@ -74,6 +74,8 @@ def __init__(
         transformation_mlp_layers: List[int] = [128],
         transformation_mlp_activation_fun: str = "ReLU",
         transformation_mlp_dropout_rate: float = 0.2,
+        transformation_mlp_result_lower_bound: Optional[int] = None,
+        transformation_mlp_result_upper_bound: Optional[int] = None,
```

Comment:
Why `int`s instead of `float`s?

Reply:
No good reason, just my brain being slow about it...